server/db/db.go

```diff
@@ -0,0 +1,31 @@
+package db
+
+import (
+	"context"
+	"log"
+	"os"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var Ctx = context.TODO()
+
+func DB() *mongo.Database {
+	clientOptions := options.Client().ApplyURI(os.Getenv("MONGODB_URI"))
+
+	client, err := mongo.Connect(Ctx, clientOptions)
```

Comment:
The mongo `client` here should be closed by calling `client.Disconnect()` somewhere, otherwise we might have a leak here.